### ISSUE TITLE
option to always show punycode encoding of international domain names

### ISSUE THREAD

**Issue by diracdeltas** [brave/brave-browser#17232]
in light of these recent phishing campaigns targeting brave and cryptocurrency users [1], we should consider adding a preference to always show punycode instead of IDN. could perhaps look at what ungoogled chromium does: https://github.com/brave/brave-browser/issues/1431#issuecomment-444166133

[1] https://twitter.com/martijn_grooten/status/1420480947863048192

**Comment by diracdeltas** [comment]
https://github.com/Eloston/ungoogled-chromium/blob/745d63aeb7ea600ae118f614fcf5095ccde5e003/patches/extra/ungoogled-chromium/add-flag-to-force-punycode-hostnames.patch < we'd want it as a pref in brave://settings instead of a command line flag

**Comment by LordRishav** [comment]
There are multiple flags I miss after switching from ungoogled-chromium to Brave, like this one, `--disable-search-engine-collection` to disable automatic search engine scraping, `--popups-to-tabs` (favourite) to make popups open in tabs instead of seperate windows and `SetIpv6ProbeFalse` to disable IPv6

**Comment by Tanya1910** [comment]
Hey, I have a fair understanding of this issue, can I work on this?

**Comment by koko1928** [comment]
> in light of these recent phishing campaigns targeting brave and cryptocurrency users [1], we should consider adding a preference to always show punycode instead of IDN. could perhaps look at what ungoogled chromium does: [#1431 (comment)](https://github.com/brave/brave-browser/issues/1431#issuecomment-444166133)
> 
> [1] https://twitter.com/martijn_grooten/status/1420480947863048192

I agree. I think that the damage can be reduced a little by displaying a warning dialog. Please let us know your thoughts.

**Comment by snowbound** [comment]
I assume nothing has become of this after 2 years

**Comment by kartibrown** [comment]
This should really be a feature...
